### ISSUE TITLE
Adding page numbers to citation

### ISSUE THREAD

**Issue by saltiredj** [dsifford/academic-bloggers-toolkit#269]
The new version of the plugin which allows editing is fantastic! Thanks!

Often, a reference needs to include a page number (e.g., Smith 2009:15 or Smith 2009 p.15) but at the moment the plugin doesn't allow for this. It would be helpful to be able to add a suffix to the year in order to do this. I've included an example of how this works using Pages (Mac) and Endnote; the same concept works for Microsoft Word, Scrivener, etc.

Just a suggestion for the next version. :-)

Again, many thanks!
![screenflow mp4](https://cloud.githubusercontent.com/assets/24391106/21125678/3eb31656-c0df-11e6-8eb2-b6a68f719f5c.gif)

**Comment by dsifford** [comment]
@fbennett Looks like CSL-M is capable of appending such extensions for this type of information. How does one go about enabling this?

**Comment by dsifford** [comment]
@fbennett Did you get my mention above? Is there any documentation on this anywhere that I may have overlooked?

**Comment by saltiredj** [comment]
Sorry. I just got this email now. Have been off the grid. Is there something I need to read/check?

Sent from my iPhone

On 4 Jan 2017, at 01:09, Derek Sifford <notifications@github.com<mailto:notifications@github.com>> wrote:


@fbennett<https://github.com/fbennett> Did you get my mention above? Is there any documentation on this anywhere that I may have overlooked?

-

**Comment by dsifford** [comment]
@saltiredj Nope, just waiting to hear back from @fbennett on how to implement with his processor.

**Comment by fbennett** [comment]
(Bennett here - finally responded to Derek, very sorry for the long delay.)

**Comment by jugdemon** [comment]
I would like to piggyback this issue: I am currently adding pages to specific citations and I would rather not cite the same article 5 times with different pages. Would it be possible to make such page citations a local edit to a specific inserted citation?

**Comment by dsifford** [comment]
@jugdemon Yep, what you're describing is what @saltiredj is suggesting. I'll look into this when I get some spare time.

**Comment by jugdemon** [comment]
No worries, for now I just write the page in extra-parenthesis, but an in-built solution will be nicer.

Thanks for working so dedicated on this, this is most certainly the best-maintained wordpress plugin I came across so far.

**Comment by mredaelli** [comment]
I'd also love to have this feature :+1:

**Comment by mjluser1** [comment]
I also am doing a work-around by listing the page numbers in brackets with the citation.  This feature would be great, but I understand the bugs need to be addressed first.  Thanks for the plugin!
